backend/internal/types/occurrence.go

```diff
@@ -3,82 +3,89 @@ package types
-	IsCanonical         bool              `json:"is_canonical"`
-	AaChange            string            `json:"aa_change,omitempty"`
-	RsNumber            string            `json:"rsnumber,omitempty"`
-	PickedConsequences  JsonArray[string] `gorm:"type:json" json:"picked_consequences" validate:"required"`
-	TranscriptId        string            `json:"transcript_id,omitempty"`
-	MaxImpactScore      int               `json:"max_impact_score" validate:"required"`
-	HasInterpretation   bool              `json:"has_interpretation" validate:"required"`
+	SeqId                      int               `json:"seq_id" validate:"required"`
+	TaskId                     int               `json:"task_id" validate:"required"`
+	Chromosome                 string            `json:"chromosome" validate:"required"`
+	Start                      int64             `json:"start" validate:"required"`
+	LocusId                    string            `json:"locus_id" validate:"required"`
+	GenotypeQuality            int32             `json:"genotype_quality" validate:"required"`
+	Filter                     string            `json:"filter,omitempty"`
+	Zygosity                   string            `json:"zygosity" validate:"required"`
+	PfWgs                      float64           `json:"pf_wgs" validate:"required"`
+	PcWgs                      int               `json:"pc_wgs,omitempty"`
+	PnWgs                      int               `json:"pn_wgs,omitempty"`
+	GnomadV3Af                 float64           `json:"gnomad_v3_af" validate:"required"`
+	Hgvsg                      string            `json:"hgvsg" validate:"required"`
+	OmimInheritanceCode        JsonArray[string] `gorm:"type:json" json:"omim_inheritance_code,omitempty"`
+	AdRatio                    float32           `json:"ad_ratio" validate:"required"`
+	VariantClass               string            `json:"variant_class" validate:"required"`
+	VepImpact                  VepImpact         `json:"vep_impact,omitempty" enums:"MODIFIER,LOW,MODERATE,HIGH"`
+	Symbol                     string            `json:"symbol,omitempty"`
+	Clinvar                    JsonArray[string] `gorm:"type:json" json:"clinvar,omitempty"`
+	IsManeSelect               bool              `json:"is_mane_select"`
+	IsManePlus                 bool              `json:"is_mane_plus"`
+	IsCanonical                bool              `json:"is_canonical"`
+	AaChange                   string            `json:"aa_change,omitempty"`
+	RsNumber                   string            `json:"rsnumber,omitempty"`
+	PickedConsequences         JsonArray[string] `gorm:"type:json" json:"picked_consequences" validate:"required"`
+	TranscriptId               string            `json:"transcript_id,omitempty"`
+	MaxImpactScore             int               `json:"max_impact_score" validate:"required"`
+	HasInterpretation          bool              `json:"has_interpretation" validate:"required"`
+	ExomiserMoi                string            `json:"exomiser_moi"`
```

Comment:
add validate:"required" if they are in occurrences default fields
or omitempty if they are user defined fields